src/containers/my-resources/attachments-container/AttachmentsContainer.constants.tsx

```diff
@@ -0,0 +1,56 @@
+import Typography from '@mui/material/Typography'
+
+import IconExtentionWithTitle from '~/components/icon-extention-with-title/IconExtentionWithTitle'
+import { getFormattedDate } from '~/utils/helper-functions'
```

Comment:
component import must be separate from these bottom imports

Reply:
done!